PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -6,14 +6,15 @@
 
   <description><![CDATA[
       <html>
-        <p>These plugins allow Java developers, Azure HDInsight developers and Apache Spark on SQL Server users to easily create, develop, configure, test, and deploy highly available and scalable Java web apps and Apache Spark/Hadoop jobs to Azure from IntelliJ on all supported platforms.</p>
+        <p>The plugin allows Java developers to easily develop, configure, test, and deploy highly available and scalable Java web apps. It also supports Azure Synapse data engineers, Azure HDInsight developers and Apache Spark on SQL Server users to create, test and deploy Apache Spark/Hadoop jobs to Azure from IntelliJ on all supported platforms.</p>
```

Comment:
deploy Apache Spark/Hadoop jobs -> submit Apache Spark/Hadoop jobs ?

Reply:
Good catch! Will fix it in next commit.